tests/TestCase/Error/DebuggerTest.php

```diff
@@ -73,13 +75,12 @@ public function tearDown(): void
      */
     public function testDocRef()
     {
-        $this->skipIf(
-            defined('HHVM_VERSION'),
-            'HHVM does not output doc references'
-        );
         ini_set('docref_root', '');
         $this->assertEquals(ini_get('docref_root'), '');
-        new Debugger();
+        // Force a new instance.
+        Debugger::getInstance(TestDebugger::class);
```

Comment:
This is odd looking code lol.

Reply:
Yes. Debugger::getInstance() checks if the old and new instance are the same class. If they are it doesn't build a new instance. Annoying.